airflow/api_connexion/endpoints/pool_endpoint.py

```diff
@@ -114,7 +114,10 @@ def patch_pool(
         update_mask = [i.strip() for i in update_mask]
         _patch_body = {}
         try:
-            update_mask = [
+            # MyPy infers a List[Optional[str]]  type here but it should be a List[str]
```

Comment:
Not sure why this one started to happen (mypy quirks) but the error signalled by mypy here makes no sense.